test/e2e/buildx_inspect_test.go

```diff
@@ -0,0 +1,209 @@
+//go:build linux || freebsd
+
+package integration
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/containers/podman/v5/cmd/podman/registry"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman buildx inspect", func() {
+	It("podman-remote buildx inspect", func() {
+		if !registry.IsRemote() {
+			Skip("This test is only applicable for remote podman")
+		}
+
+		session := podmanTest.PodmanExitCleanly("buildx", "inspect")
+		output := session.OutputToString()
+
+		infoSession := podmanTest.PodmanExitCleanly("info", "--format", "{{.Host.OS}}/{{.Host.Arch}}")
+		expectedNativePlatform := strings.TrimSpace(infoSession.OutputToString())
+
+		Expect(output).To(MatchRegexp(`Platforms:\s+.*\b` + regexp.QuoteMeta(expectedNativePlatform) + `\b`))
+	})
+
+	It("podman-remote buildx inspect --bootstrap", func() {
+		if !registry.IsRemote() {
+			Skip("This test is only applicable for remote podman")
+		}
+
+		session := podmanTest.PodmanExitCleanly("buildx", "inspect", "--bootstrap")
+		output := session.OutputToString()
+
+		infoSession := podmanTest.PodmanExitCleanly("info", "--format", "{{.Host.OS}}/{{.Host.Arch}}")
+		expectedNativePlatform := strings.TrimSpace(infoSession.OutputToString())
+
+		Expect(output).To(MatchRegexp(`Platforms:\s+.*\b` + regexp.QuoteMeta(expectedNativePlatform) + `\b`))
+	})
+
+	It("podman buildx inspect", func() {
```

Comment:
Great that you introduced tests for this change :-).

I think there's a some duplicated code here:

- Native platform parsing
- Emulated Architectures parsing
- Expected Platform parsing

Maybe you could move these to separated functions and reuse them to make the tests easier to read and also edit in the future if we change the way how this command behaves.